File: async-scala-client/src/main/scala/io/swagger/client/model/Fuel_cide.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class Fuel_cide (
  baseCalcCIDE: Double,  // percentage of natural gas (GLP), this decimal 12 integers and 0 to 4 decimals
rateCIDE: Double,  // percentage of natural gas (GLP), this decimal 11 integers and 0 to 4 decimals - R$
valueCIDE: Double  // percentage of natural gas (GLP), this decimal 13 integers and 2 decimals - R$
)

File: javascript-closure-angular-client/API/Client/FuelCide.js
goog.provide('API.Client.Fuel_cide');

/**
 * @record
 */
API.Client.FuelCide = function() {}

/**
 * percentage of natural gas (GLP), this decimal 12 integers and 0 to 4 decimals
 * @type {!number}
 * @export
 */
API.Client.FuelCide.prototype.baseCalcCIDE;

/**
 * percentage of natural gas (GLP), this decimal 11 integers and 0 to 4 decimals - R$
 * @type {!number}
 * @export
 */
API.Client.FuelCide.prototype.rateCIDE;

/**
 * percentage of natural gas (GLP), this decimal 13 integers and 2 decimals - R$
 * @type {!number}
 * @export
 */
API.Client.FuelCide.prototype.valueCIDE;

